report/main.qmd

```diff
@@ -0,0 +1,103 @@
+---
+title: 
+  "Investigating the efficacy of consumer interventions on sales of zero-calorie beverages"
+format: 
+  pdf:
+    fontsize: 11pt
+date:
+  2024-03-01
+editor: 
+  visual
+bibliography: 
+  ../docs/references.bib
+csl: 
+  ../docs/elsevier-vancouver.csl
+author: 
+- Christine Chuong (20124798)
+- Giuliano Cruz (79247979)
+- Joey Hotz (18461400)
+include-in-header: 
+  text: |
+    \usepackage{authblk}
+    \pagenumbering{gobble}
+---
+
+{{< pagebreak >}}
+
+```{=tex}
+\pagenumbering{arabic}
+```
+
+# Introduction
+
+Strong evidence has linked habitual intake of sugar-sweetened beverages (SSBs) with weight gain and a higher risk of type 2 diabetes mellitus, cardiovascular diseases, and even cancer @Malik2022. Multiple policy interventions to inhibit consumption of SSBs have been proposed @Krieger2021, including taxation of SSBs, limiting access to SSBs in schools and healthcare facilities, and adding informative package labelling. As policymakers create potential interventions to limit SSB consumption, there is a need for data-driven methods of determining their efficacy.
+
+This study aims to evaluate five interventions which intend to shift consumer behaviour towards zero-calorie beverages. These interventions include three information-based interventions and two financial interventions.
```

Comment:
I think the primary objective is to compare two interventions (labelling calorie content) and then price discounts are a secondary research question